doc/en/user/source/community/wfs-freemarker/configuration.rst

```diff
@@ -0,0 +1,130 @@
+.. _community_wfsfreemarker_config:
+
+WFS FreeMarker Extension configuration
```

Comment:
I'm confused by this section. The code is using the GeoServer GetFeatureInfo machinery, so there is nothing new here in terms of configuration, it's the same files as the GetFeatureInfo... one can simply link to that documentation.

The screenshots are also confusing, the OpenLayers preview client is using GetFeatureInfo rather than GetFeature, so it's not using this module's functionality at all.

Suggestions:
1) Link to the [GFI documentation](https://docs.geoserver.org/main/en/user/tutorials/GetFeatureInfo/html.html) for details on how to configure, rather than creating a new example
2) Use an actual GetFeature as an example of usage, with outputs with vanilla templates, and then with the [customized ones](https://docs.geoserver.org/main/en/user/tutorials/GetFeatureInfo/html.html#custom-templates)

Reply:
Done